src/tracks/playabletrack/wavetrack/ui/WaveformView.cpp

```diff
@@ -1143,21 +1132,24 @@ BEGIN_POPUP_MENU(WaveColorMenuTable)
       OnInstrument4ID = hostTable.ReserveId();
    } );
 
-   AppendRadioItem( "Instrument1", OnInstrument1ID,
-      GetWaveColorStr(0), POPUP_MENU_FN( OnWaveColorChange ), fn );
-   AppendRadioItem( "Instrument2", OnInstrument2ID,
-      GetWaveColorStr(1), POPUP_MENU_FN( OnWaveColorChange ), fn );
-   AppendRadioItem( "Instrument3", OnInstrument3ID,
-      GetWaveColorStr(2), POPUP_MENU_FN( OnWaveColorChange ), fn );
-   AppendRadioItem( "Instrument4", OnInstrument4ID,
-      GetWaveColorStr(3), POPUP_MENU_FN( OnWaveColorChange ), fn );
+   for (size_t ii = 0; ii < 3; ++ii) {
+      AppendRadioItem(
+         wxString::Format("Instrument%d", ii),
```

Comment:
In the original the popup menu items starting from Instrument1 to Instrument4.  This has been refactored to have one less menu item and start from Instrument0, is this intentional?

Reply:
Aha, good catch.  Unintended change of menu labels.  I should insert `+ 1`

Reply:
Rebased onto recent master; the +1 fix is now squashed into the last commit